### ISSUE TITLE
jsx-a11y/no-noninteractive-tabindex should not flag `tooltip` role

### ISSUE THREAD

**Issue by jessebeach** [jsx-eslint/eslint-plugin-jsx-a11y#576]
```
<div
  aria-describedby={labelID}
  role="tooltip"
  tabIndex={0}>
```

A tooltip is meant to be tabbable without being actionable.

**Comment by jessebeach** [comment]
Hmm, here's better guidance. The `tooltip` role is non-interactive and it does not receice focus.

```
Tooltip widgets do not receive focus. A hover that contains focusable elements can be made using a non-modal dialog.
```

https://www.w3.org/TR/wai-aria-practices-1.1/#tooltip

**Comment by jessebeach** [comment]
There's nothing to do here. Tooltips should not receive focus. They are content. At Meta, we associate them with their related element by using aria-describedby on the source element and this points to the tooltip's ID. Only interactive element should receive focus and tooltips are static content. If your tooltip contains interactive content, then it is a dialog and not a tooltip.
